README.md

```diff
@@ -27,3 +27,9 @@ In your local `.eslintrc`
   "extends": ["@logojoy", "@logojoy/react"]
 }
 ```
+
```

Comment:
I would a note on how to consume the `@logojoy/mocha` package

Reply:
Thanks!